src/frontend/src/redux/groups/actions.ts

```diff
@@ -0,0 +1,149 @@
+import { Dispatch } from 'redux';
+import { LOAD_GROUPS, Group } from './types';
+
+export const loadGroups = () => (dispatch: Dispatch) => {
+  return new Promise(resolve => {
```

Comment:
Not that it really matters since this code will be replaced with the actual request, but since I thought of it lol the Promise should probably be resolved.